docs/howto/error-reporting.txt

```diff
@@ -255,13 +255,6 @@ following methods:
     Returns ``True`` to activate the filtering operated in the other methods.
     By default the filter is active if :setting:`DEBUG` is ``False``.
 
-.. method:: SafeExceptionReporterFilter.get_request_repr(request)
```

Comment:
I'm suggesting refactoring of the exception reporter in a way that gives developers better control over the report, so they can sanitize anything they need. They are the only one who can decide which data are sensitive in their case. 

Addition of a method for sanitizing META arguments only solves one particular problem, but other problems may be found out which would require addition of other methods for particular parts of error report. I wouldn't oppose to the ticket to add a `get_meta_parameters()`, but I'd rather see a more sophisticated solution.

If the developers are given control over the context and template of the error report they will be able on one hand sanitize the details of the report to their needs and on the other hand include any additional data they consider important to debug the problem. These features are repeatedly requested
 - display username in error report [ticket 25037](https://code.djangoproject.com/ticket/25037)
 - sanitize exception message [ticket 23951](https://code.djangoproject.com/ticket/23951)
 - existing ticket to sanitize META arguments [ticket 23004](https://code.djangoproject.com/ticket/23004)

If you agree to give developers greater control over the error reports I'll gladly create a ticket with proposal which will have minimal impact on existing code and provide a patch.